src/PollutantBarComponent.js

```diff
@@ -112,10 +112,16 @@ function PollutantBarComponent(props) {
     };
 
     const getNumericalVal = () => {
+        if (pollutant === 'PM25' || pollutant === 'PM10') {
+            // TODO: modularize with correctionFactor prop in Utils.js
```

Comment:
I'm cool with not doing this change right now, but here's an [Issue](https://github.com/airpartners/aq-web-client/issues/50) about this TODO.